Examples/Algorithms/Digitization/src/HitSmearing.cpp

```diff
@@ -72,14 +72,19 @@ ActsExamples::ProcessCode ActsExamples::HitSmearing::execute(
     const Acts::Surface* surface = is->second;
     for (const auto& hit : moduleHits) {
       // transform global position into local coordinates
-      Acts::Vector2D pos(0, 0);
-      surface->globalToLocal(ctx.geoContext, hit.position(),
-                             hit.unitDirection(), pos);
+      auto lpResult = surface->globalToLocal(ctx.geoContext, hit.position(),
+                                             hit.unitDirection());
+      Acts::Vector2D lp{0., 0.};
+      if (not lpResult.ok()) {
+        ACTS_WARNING("Global to local transformation did not succeed.");
```

Comment:
What if we upgrade to ERROR?

Reply:
I would still prefer to fail the algorithm altogether. If you are running many jobs in parallel this can easily get missed or be 100 down in the history. If we see that this occurs too often and make the example runs unusable we can switch back to an error.